agent/engine/docker_image_manager_integ_test.go

```diff
@@ -378,8 +378,12 @@ func TestImageWithSameNameAndDifferentID(t *testing.T) {
 	imageState2.PulledAt = imageState2.PulledAt.Add(-19 * time.Minute)
 	imageState3.PulledAt = imageState3.PulledAt.Add(-18 * time.Minute)
 
-	// Verify Task is stopped
-	verifyTaskIsStopped(taskEvents, task1, task2, task3)
+	go discardEvents(taskEvents)
```

Comment:
After looking at the other tests, there are actually some benefits to use `verifyTaskIsStopped`. If we want to use this for other place, then we need to call `discardEvent` for both `taskEvent` and `containerEvents`. Let me know if you still think it's better to get rid of `verifyTaskIsStopped()`. Also we need another similar function for `verifyTaskIsRunning` to get rid of all, since there are some tests used both `verifyTaskIsRunning` and `verifyTaskIsStopped` in which case `discardEvents` will cause `verifyTaskIsRunning` to fail. If you still think we should do this, then I will create a separate PR.